packages/nextjs/src/create-server-side-client.ts

```diff
@@ -0,0 +1,44 @@
+import Cookies from 'cookies'
+import { GetServerSidePropsContext } from 'next'
+import { StateFrom } from 'xstate'
+import { waitFor } from 'xstate/lib/waitFor'
+
+import { AuthMachine } from '@nhost/core'
+import { NhostClient } from '@nhost/nhost-js'
+
+/**
+ * Creates an Nhost client that runs on the server side.
+ * It will try to get the refesh token in cookies, or from the request URL
+ * If a refresh token is found, it uses it to get an up to date access token (JWT) and a user session
+ * This method resolves when the authentication status is known eventually
+ * @param backendUrl
+ * @param context
+ * @returns
```

Comment:
Could you please provide some description for this tag please?

It would be displayed in a weird way if left as is:
https://github.com/nhost/nhost/pull/459/files#diff-0f7398ca23f3fcc1358ae4887c30e6c5351df493f32b8f44c021d8333f8b7d97R15